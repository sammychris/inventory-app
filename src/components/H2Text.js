import styled from 'styled-components';


const H2Text = ({children, align}) => {
    return (
        <Container align={align}>
            <H2>
                {children}
            </H2>
        </Container>
    );
  }


export default H2Text;

const H2 = styled.h2`
  font-size: 24px;
`;

const Container = styled.div` 
  text-align: ${(props) => props.align};
`;
