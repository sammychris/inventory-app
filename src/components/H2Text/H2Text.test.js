import { shallow } from "enzyme";
import H2Text from "./H2Text";

const wrapper = shallow(<H2Text />);

test("expect to render a H2Text", () => {
  expect(wrapper.length).toEqual(1);
});

test("expect to make a snap shot to H2Text component", () => {
  expect(wrapper).toMatchSnapshot();
});