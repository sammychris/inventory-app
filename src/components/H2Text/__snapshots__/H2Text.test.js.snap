// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`expect to make a snap shot to H2Text component 1`] = `ShallowWrapper {}`;
